pages/_app.tsx

```diff
@@ -12,6 +12,7 @@ import '@fontsource/noto-sans/700.css'; // Noto sans bold
 const apolloClient = new ApolloClient({
   uri: '/api/graphql',
   cache: new InMemoryCache(),
+  connectToDevTools: true,
```

Comment:
Just wondering, what does this do?

Reply:
😅 i added apollo dev tools while debugging which needed this line and forgot to remove it, nice catch!